src/modules/remove_dom.js

```diff
@@ -0,0 +1,5 @@
+export default function removeAllChildElements(parent) {
+  while (parent.firstChild) {
+    parent.removeChild(parent.firstChild);
+  }
+}
```

Comment:
- [ ] Kindly use the arrow function instead of regular functions. [why you should arrow functions.](https://betterprogramming.pub/why-javascript-arrow-functions-are-useful-aa023bad403a)